services/vision/mlvision/ml_model_test.go

```diff
@@ -92,7 +92,7 @@ func TestAddingIncorrectModelTypeToModel(t *testing.T) {
 	test.That(t, classifier, test.ShouldNotBeNil)
 
 	err = checkIfClassifierWorks(ctx, classifier)
-	test.That(t, err, test.ShouldNotBeNil)
+	test.That(t, err, test.ShouldBeNil)
```

Comment:
I think this change makes sense. That first tensor that comes out of a detector _could_ totally be a bunch of probabilities (or logits). It's not, but it totally could be.

Reply:
Yeaaaah I take this back.